.github/workflows/benchmark-pipeline.yaml

```diff
@@ -103,7 +103,8 @@ jobs:
 
           kubectl wait pod \
           --all \
-          --for=condition=Ready
+          --for=condition=Ready \
+          --namespace=falco # TODO: Revert to "benchmark" this after merging https://github.com/falcosecurity/cncf-green-review-testing/pull/22
```

Comment:
Instead of hardcoding we need to get the namespace name from the project.json

Reply:
We don't need to do that at the moment, IIUC we're using a generic `benchmark` namespace for all right now. I don't see us running 2 benchmark tests simultaneously in the future either